<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Ressources.qrc">
    <normaloff>:/images/QCeCWriter.png</normaloff>:/images/QCeCWriter.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>491</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelDossierDefautImages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dossier pas défaut des images :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDossierTemporaire">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dossier temporaire :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCommandeNavigateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Commande pour lancer le navigateur  :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDossierSauvegardes">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dossier contenant les sauvegardes :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPseudoWordpress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Pseudo Wordpress : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDossierDistantImages">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dossier distant contenant les images :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEditeurImages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Editeur avancé d'images :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdresseSite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Adresse du site :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAddressePublication">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Adresse de publication :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSystemeExploitation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Système d'exploitation :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDossierDefautImages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonModifDefautImages">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonModifDossierTemp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier ...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDossierTemp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCommandeNavigateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonModifCommandeNavigateur">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDossierSauvegardes">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonModifDossierSauvegardes">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier ...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPseudoWordpress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEditeurImages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonModifEditeurImages">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDossierDistantImages">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAdresseSite">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>511</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAdressePublication">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>461</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLinuxSlack">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>392</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linux Slackware</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLinuxAutre">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linux (autre)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLinuxUbuntu">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>392</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linux Ubuntu / Debian like</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonWindows">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Windows</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOptionsSupprimerDonneesPerso">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer les données personnelles (identifiants)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOptionsAnnuler">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOptionsValider">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDossierSauvegardes_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dictionnaire path :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDictionnairePath">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditDossierDefautImages</tabstop>
  <tabstop>lineEditDossierTemp</tabstop>
  <tabstop>lineEditCommandeNavigateur</tabstop>
  <tabstop>lineEditDossierSauvegardes</tabstop>
  <tabstop>lineEditPseudoWordpress</tabstop>
  <tabstop>lineEditDictionnairePath</tabstop>
  <tabstop>lineEditEditeurImages</tabstop>
  <tabstop>lineEditDossierDistantImages</tabstop>
  <tabstop>lineEditAdresseSite</tabstop>
  <tabstop>lineEditAdressePublication</tabstop>
  <tabstop>radioButtonLinuxSlack</tabstop>
  <tabstop>radioButtonLinuxUbuntu</tabstop>
  <tabstop>radioButtonLinuxAutre</tabstop>
  <tabstop>radioButtonWindows</tabstop>
  <tabstop>buttonOptionsSupprimerDonneesPerso</tabstop>
  <tabstop>buttonOptionsValider</tabstop>
  <tabstop>buttonOptionsAnnuler</tabstop>
  <tabstop>buttonModifDefautImages</tabstop>
  <tabstop>buttonModifDossierTemp</tabstop>
  <tabstop>buttonModifCommandeNavigateur</tabstop>
  <tabstop>buttonModifDossierSauvegardes</tabstop>
  <tabstop>buttonModifEditeurImages</tabstop>
 </tabstops>
 <resources>
  <include location="Ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOptionsAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
